pyston/forms.py

```diff
@@ -43,8 +43,7 @@ def _merge_from_initial(self):
         filt = lambda v: v not in self.data.keys()
         for field_name in filter(filt, self.fields.keys()):
             field = self.fields[field_name]
-
-            self.data[field_name] = field.prepare_value(self.initial.get(field_name, field.initial))
+            self.data[field_name] = self.initial.get(field_name, field.initial)
```

Comment:
ha toto tam budu muset vrátit řeším ten blank vz none a je to celkem slušný problém, probereme osobně